src/thor/route_matcher.cc

```diff
@@ -364,7 +427,7 @@ bool RouteMatcher::FormPath(const std::shared_ptr<DynamicCost>* mode_costing,
         // Update the elapsed time based on edge cost
         elapsed += mode_costing[static_cast<int>(mode)]->EdgeCost(de, end_node_tile) *
                    (end.second.first.percent_along() - edge.percent_along());
-        if (use_timestamps)
+        if (options.use_timestamps())
```

Comment:
for single line if statement, sometimes it has {} sometimes it does not, maybe it is better to enforce it with {}?